volume/btrfs/btrfs.go

```diff
@@ -317,8 +318,41 @@ func (v *BtrfsVolume) isSnapshot(rawLabel string) bool {
 	return strings.HasPrefix(rawLabel, v.getSnapshotPrefix())
 }
 
+// writeSnapshotInfo writes metadata about a snapshot
+func (v *BtrfsVolume) writeSnapshotInfo(label string, info *volume.SnapshotInfo) error {
```

Comment:
We discussed decided to defer this in favor of moving on to bigger/better changes in the DFS